src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md

```diff
@@ -2,14 +2,18 @@
 
 ## Unreleased
 
+* Added `net.host.name` and `net.host.port` dimensions to `http.server.duration`
```

Comment:
Mention something about the dimension `http.host` no longer being populated for the metric.

Reply:
my bad, this is a breaking change. Fixed